api/organisations/subscriptions/constants.py

```diff
@@ -1,10 +1,12 @@
 from enum import Enum
 
+from django.conf import settings
+
 from organisations.subscriptions.metadata import BaseSubscriptionMetadata
 
-MAX_SEATS_IN_FREE_PLAN = 1
+MAX_SEATS_IN_FREE_PLAN = settings.MAX_SEATS_IN_FREE_PLAN or 1
```

Comment:
Unless someone creates a settings file that overrides these settings to `None` (which is their own fault), I can't see any scenario where the `or 1` is needed here. I'd say we should remove.

Reply:
Yes, makes sense to remove it. do you want me to replace each occurrence of `MAX_SEATS_IN_FREE_PLAN`  with `settings.MAX_SEATS_IN_FREE_PLAN` as well?  (same for `MAX_PROJECTS_IN_FREE_PLAN`)

Reply:
Ideally we'd just use `settings.` everywhere yeah.